config/locales/en.yml

```diff
@@ -2666,6 +2666,7 @@ en:
   large: large
   last_active_with_date: "Last Active: %{date}"
   last_confirmed_observation: Last confirmed observation
+  last_edited_by: Last edited by
```

Comment:
Please add a comment explaining to translators what this means and how it's used, e.g. "label for a field listing the name of the user who last edited a record"